tree/dataframe/inc/ROOT/RDF/RMergeableValue.hxx

```diff
@@ -0,0 +1,128 @@
+#ifndef ROOT_RDF_RMERGEABLEVALUE
+#define ROOT_RDF_RMERGEABLEVALUE
+
+#include <functional>
+#include <memory>
+
+#include "RtypesCore.h"
+#include "TList.h"
+
+namespace ROOT {
+namespace RDF {
```

Comment:
`RMergeableValue` is in `ROOT::RDF` but the only way to get one is in `ROOT::Detail::RDF`. We should align the two. I think `RMergeableValue` should move to `ROOT::Detail::RDF`.

Reply:
Yes, I was planning on doing at some point sorry

Reply:
Aligned now